terraspace/app/stacks/publishing/tfvars/us-west-2/prod.tfvars

```diff
@@ -0,0 +1,15 @@
+provider_ads_bucket_name = "ipfs-advertisement"
+ads_topic_name="advertisements-topic"
+content_lambda = {
+  name              = "publishing-content"
+  metrics_namespace = "publishing-lambda-metrics"
+  image_uri         = "505595374361.dkr.ecr.us-west-2.amazonaws.com/publisher-lambda:latest"
```

Comment:
can we avoid hardcoding the registry?

Reply:
Done! Made it consistent with the indexing stack.